server/index.js

```diff
@@ -74,7 +70,7 @@ app.get('/api/cart', (req, res, next) => {
   WHERE "cartItems"."cartId" = $1
   `;
 
-  db.query(!req.session.cartId ? sqlEmpty : sqlExist, [req.session.cartId])
+  db.query(!req.session.cartId ? [] : sqlExist, [req.session.cartId])
```

Comment:
Does this actually work? I'm thinking a simple conditional will get the job done

```js
if (!req.session.cartId) {
  return res.json([])
}
```